tests/DCPS/XTypes/XTypes.mpc

```diff
@@ -0,0 +1,32 @@
+project(*Subscriber): dcps_test, dcps_rtps_udp, msvc_bigobj, dcps_transports_for_test, opendds_optional_security {
+
+  exename = subscriber
+
+  includes += .
+
```

Comment:
```suggestion
```

This and the other `includes += .` are not needed if everything's in the same directory.